src/stdatamodels/asdf_in_fits.py

```diff
@@ -5,7 +5,34 @@
 
 from . import fits_support
 
-__all__ = ["write", "open"]
+__all__ = ["write", "open", "to_hdulist"]
+
+# This API is intended to replace the removed asdf.AsdfInFits
+# for community (non-pipeline) usage. When considering changes
+# a wider search for usage beyond pipeline code and documentation
+# is recommended as well as longer deprecation periods that
+# aren't as closely linked to pipeline releases.
```

Comment:
Is it standard practice to have this in `#` comments instead of triple-quoted at the top of the file?

Reply:
Thanks. I used a comment here so that it doesn't get auto-included in the RTD here:
https://stdatamodels--567.org.readthedocs.build/en/567/api.html#module-stdatamodels.asdf_in_fits
(which I think would be the case for a module level docstring/triple-quoted text at the top fo the file).